<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skills Manager 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>321</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="expandAllButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Expand All</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select semester / course</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select semester / course</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select semester / course</string>
    </property>
   </widget>
   <widget class="QListWidget" name="compListWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="prereqListWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="toggleReasonerCheckBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Synchronize Reasoner</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lackSkillsListWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select semester / course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showCompButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Competences</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showChartButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowChart</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsdgds">
   <property name="text">
    <string>sdgds</string>
   </property>
  </action>
  <action name="actionhelpaction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>helpaction</string>
   </property>
  </action>
  <action name="showComp">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="analyzeCurriculum">
   <property name="text">
    <string>Analyze curriculum</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
